semantic/tasks/config/project/install.js

```diff
@@ -0,0 +1,758 @@
+/*******************************
+            Set-up
+*******************************/
+
+var
+  fs             = require('fs'),
+  path           = require('path'),
+  defaults       = require('../defaults'),
+  release        = require('./release'),
+
+  requireDotFile = require('require-dot-file')
+;
+
+/*******************************
+          When to Ask
+*******************************/
+
+/* Preconditions for install questions */
+
+var when = {
+
+  // path
+  changeRoot: function(questions) {
+    return (questions.useRoot !== undefined && questions.useRoot !== true);
+  },
+
+  // permissions
+  changePermissions: function(questions) {
+    return (questions.changePermissions && questions.changePermissions === true);
+  },
+
+  // install
+  hasConfig: function() {
+    return requireDotFile('semantic.json', process.cwd());
+  },
+
+  allowOverwrite: function(questions) {
+    return (questions.overwrite === undefined || questions.overwrite == 'yes');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)